bin/improver-temp-lapse-rate

```diff
@@ -94,14 +102,24 @@ def main():
     orography_cube = load_cube(args.orography_filepath)
     land_sea_mask_cube = load_cube(args.land_sea_mask_filepath)
 
-    result = LapseRate(
-        max_height_diff=args.max_height_diff,
-        nbhood_radius=args.nbhood_radius,
-        max_lapse_rate=args.max_lapse_rate,
-        min_lapse_rate=args.min_lapse_rate).process(temperature_cube,
-                                                    orography_cube,
-                                                    land_sea_mask_cube)
-
+    if args.return_dalr:
+        result = temperature_cube.copy(
+            data=np.full_like(temperature_cube.data, U_DALR.points[0]))
+        result.rename('temperature_lapse_rate')
+        result.units = U_DALR.units
+        result.attributes['source'] = 'IMPROVER'
+        result.attributes['institution'] = 'Met Office'
+        add_history_attribute(result, ['IMPROVER'])
```

Comment:
Having had a quick look at the `add_history_attribute` function, and previous usage of this function, this could potentially be updated, as in other places, this function is called as `add_history_attribute(result, ['add', 'IMPROVER'])`, however, the `'add'` is immediately stripped out. It might be worth amending this function to remove usage of the `'add'` keyword, as it always adds a history attribute anyway.

Reply:
OK, will do.